projects/control-service/projects/helm_charts/pipelines-control-service/templates/alertmanager.yaml

```diff
@@ -0,0 +1,53 @@
+{{- /*
+  Copyright 2021-2023 VMware, Inc.
+  SPDX-License-Identifier: Apache-2.0
+ */}}
+
+{{- if .Values.alertmanager.enabled }}
+apiVersion: v1
+kind: List
+metadata:
+  resourceVersion: ""
```

Comment:
Those two lines should not be needed. The resourceVersion is usually managed by Kubernetes itself. 


I am guessing you generated this using kubectl get -o yaml or similar command  and edited it ?? You've missed to clean up a few things.